src/main/play-doc/developer/AkkaAndPlay.md

```diff
@@ -186,6 +186,22 @@ class MyCustomApplicationLoader extends ApplicationLoader {
 }
 ```
 
+### Controlling startup signalling
+
+The Play flavors of conductr-bundle-lib automatically signal that your application has started with the `StatusService`. In the event that you want finer control over signalling then you can provide your own lifecycle handler. Use [the conductr-bundle-lib one[(https://github.com/typesafehub/conductr-lib/blob/master/play25-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/play/api/ConductRLifecycle.scala)] as a reference. You must also disable the existing lifecycle module and of course register your own. To disable the existing one:
```

Comment:
Typo in this line (the closing square bracket is the wrong way, so the link doesn't resolve).

But in general, this answers my question -- thanks...